### ISSUE TITLE
Not installing from azzyaddons.github.io repository

### ISSUE THREAD

**Issue by mm1992** [azzy9/plugin.git.browser#6]
Your Environment
* Plugin Version: repository.azzyaddons v1.0.1
* Kodi Version: 19.3.0
* Operating System: Win 10

Issue

Hello,
the addon cannot be installed by AzzyAddons repository.
Here is the log:
`2024-12-04 20:02:54.798 T:204     ERROR <general>: CAddonInstallJob[plugin.git.browser]: failed to download special://home/addons/packages/plugin.git.browser-2.0.0~alpha.zip
2024-12-04 20:02:58.922 T:7832  WARNING <general>: CGUIWindowManager - CGUIWindowManager::HandleAction - ignoring action 107, because topmost modal dialog closing animation is running
2024-12-04 20:02:58.981 T:204     ERROR <general>: Skipped 1 duplicate messages..
2024-12-04 20:02:58.981 T:204     ERROR <general>: CCurlFile::Open failed with code 404 for https://raw.githubusercontent.com/azzyaddons/azzyaddons.github.io/master/plugins/plugin.git.browser/plugin.git.browser-2.0.0~alpha.zip:`
For unknown reasons, the only version possible to install via the repository is 2.0.0~alpha, while on GH there is a regular 2.0.0.

**Comment by azzy9** [comment]
My bad, it should be fixed now.

Thank you for reporting

**Comment by mm1992** [comment]
Thank you. Now it has installed well.

=======

_After installation, I am just testing this addon and currently I've managed to install one addon via your Git Browser.
In the beginning, I was not sure if there was still sth wrong, because for the first time while searching I got API Rate Exceeded error, but then I included "personal access token" from my own account and after that it was OK. Perhaps including the token is required to work which I hadn't notice._
